config/systemjs.config.js

```diff
@@ -12,12 +12,31 @@ System.config({
 
   packages: {
     "src": {
+      "main": "./main.ts",
+      "meta": {
+        "*.ts": {
+          "loader": "ts"
+        }
+      }
+    },
+    "src/lib": {
+      "defaultExtension": "ts",
+      "meta": {
+        "*.ts": {
+          "loader": "ts"
+        }
+      }
+    },
+    "src/app": {
```

Comment:
ho boy. couldn't figure out how to get systemjs to not add a `.ts` to the filenames without doing this.